taqueria-plugin-ipfs-pinata/README.md

```diff
@@ -1,19 +1,17 @@
 # IPFS Pinata Plugin
 
-This is a plugin developed for Taqueria built on NodeJS using the Taqueria Node SDK and distributed via NPM
-
 The IPFS Pinata plugin provides a `publish` task to upload metadata, media, or other files to IPFS via Pinata
 
 The IPFS Pinata plugin also provides a `pin` task to link any existing files on Pinata to your Pinata account
 
 
-
+ 
 ## Requirements
 
 - Node.js v16.3 or later
 - Docker v0.8.6 or later
 - A [Pinata](https://app.pinata.cloud) account
-- A Pinata JWT Token added to the project
+- A Pinata JWT Token added to a `.env` file in the project folder
```

Comment:
An, not a: 
`A Pinata JWT Token added to an `.env` file in the project folder`

Reply:
I put "a" because I always say this as dot-env which starts with a consonant